### PR TITLE
update mdsd for aarch64 patch

### PR DESCRIPTION
**Pull request by pfrcks**
- Update mdsd from 1.29.4 to 1.29.7 to address aarch64 crashes

### CODE REVIEW

File: kubernetes/linux/setup.sh [hunk 1]
```diff
@@ -37,7 +37,7 @@ mv /usr/lib/ruby/gems/3.1.0/specifications/default/uri-0.11.0.gemspec /usr/lib/r
 gem uninstall time --version 0.2.0
 gem uninstall uri --version 0.11.0
 
-sudo tdnf install -y azure-mdsd-1.29.4
+sudo tdnf install -y azure-mdsd-1.29.7
```
Comment (ganga1980) [inline kubernetes/linux/setup.sh:40]: 1. can you please double check this version works on both ARM64 and x64 nodes?
2.  Lets ask the MDSD team to have the release notes for this version so we and other team can refer the release notes for this. In the past, AKS team indicated that, we were using not released version by looking at MDSD release notes.

Comment (pfrcks) [reply kubernetes/linux/setup.sh:40]: 1) confirmed with Aaron that it's for both
2) will be live in an hour or so

**Review by jatakiajanvi12 — COMMENTED** [review]
Do we need to test for cpu/memory usage  for this new version? Also make sure there isn't any other regression?

**Comment by pfrcks** [comment]
compares 3.1.17 with this PR. later values are for this PR changes
legacy
legacy_cpu
legacy_mem

msi
msi_cpu
msi_mem

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**